docs/guides/migrating-to-7.0.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+title: Migrating to v7.0 - Sinon.JS
+breadcrumb: migrating to 7.0
+---
+
+For upgrading to new Sinon 7 there are no known major breaking Changes except **negative ticks** are not allowed in `Sinon@7` due to update to lolex 3. So you cannot use negative values in sinon.useFakeTimers().tick();
```

Comment:
Changes => changed

Reply:
@fatso83 I have changed the text. Thanks for reviewing